content/documentation/coffeeEditorNg/content.md

```diff
@@ -1,35 +1,45 @@
 
 <div style="text-align:center; margin-bottom:20px">
-  <img src="../../images/coffeeeditormodelhub.svg" alt="Overview of the Model Hub for the Coffee Editor NG" width="70%" />
+  <img src="../../images/modelingtools.svg" alt="Overview of the Model Hub" width="70%" />
 </div>
 
-The core of the Coffee Editor NG is a model language contribution to the Model Hub.
-With this language contribution, we register the modeling language alongside the capabilities for handling this modeling language in the Model Hub.
-The contributed Coffee Editor NG Model is defined with Langium, an open-source language toolkit for textual languages.
-The Langium-based language is then integrated into the Model Hub API with the generic *EMF Cloud Model Hub Langium* integration library, so that Model Hub clients can interact with the language on model level and benefit from Langium's efficient persistence, cross-reference management and validation mechanism.
+## Model Hub Approach
 
-In order to also simplify the editing capabilities, including undo and redo, on model level -- rather than on text level -- we use the *EMF Cloud Editing Domain*, which provies state management, as well as a command API and a command stack for arbitrary JSON models.
+This approach focuses on comprehensive model management. It is built around a central Model Hub that offers robust capabilities such as command-based editing, state management with undo/redo, and persistence. This is ideal for scenarios where a powerful environment for managing complex models is required. Detailed examples for this approach can be found in the Model Hub repository:
 
-Finally, the Coffee Editor NG language contribution adds a custom coffee-model-specific API to be used by clients in order to provide reusable model queries and manipulation functions to the clients.
+https://github.com/eclipse-emfcloud/modelhub/tree/main/examples/theia
 
-<!--
-➡️ TODO Link to try now
 
-➡️ TODO Link to source code and a good entry point to look at the language contribution
--->
+## Langium-Based DSL Approach
 
-We will provide a live demo as well as a link to GitHub as soon as the code is published.
+This approach is designed for applications that require dedicated DSL support. Leveraging Langium, it provides advanced textual language features including parsing, reference resolution, and language server integration. This path is ideal for scenarios where the development of a domain-specific language is crucial. A prominent example demonstrating this approach is the CrossModel application, available at:
 
-In the remainder of this documentation, we use the Coffee Editor NG as an example to demonstrate how certain capabilities can be customized, extended or exchanged with other implementations.
-Please use the table of contents on the left to navigate to the respective topic of interest.
+https://github.com/CrossBreezeNL/crossmodel
```

Comment:
See comment above, I think it'd be best to link here to separate EMF Cloud Getting Starting pages (or dedicated sub-sections in a common Getting Started page), which give one example or a high-level overview on what the two repositories and their examples look like and have multiple outgoing links to the respective repositories.